### PR TITLE
Add recipe for dbus-cxx

### PR DESCRIPTION
**Pull request by caiotoledo**
The library dbus-cxx is a C++ wrapper for dbus.

For more information check: https://dbus-cxx.github.io/

### CODE REVIEW

File: meta-oe/recipes-core/dbus-cxx/dbus-cxx_0.12.bb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+RDEPENDS_${PN} = "\
+	dbus \
+	libsigc++-2.0 \
+"
+
+S = "${WORKDIR}/git/"
+
+inherit pkgconfig cmake
+
+EXTRA_OECMAKE = ""
```
Comment (kraj) [inline meta-oe/recipes-core/dbus-cxx/dbus-cxx_0.12.bb:24]: This is not needed

**Review by kraj — COMMENTED** [review]
Please add it to packagegroup-meta-oe as well since it’s a new recipe

File: meta-oe/recipes-core/dbus-cxx/dbus-cxx_0.12.bb (hunk 1 above)
Comment (caiotoledo) [reply meta-oe/recipes-core/dbus-cxx/dbus-cxx_0.12.bb]: Done in 3cde237d0fb8fdc12d5e6315426faccde6a287dd

**Comment by kraj** [comment]
seeing build failures on musl

see

https://errors.yoctoproject.org/Errors/Details/539372/

**Review by kraj — CHANGES_REQUESTED** [review]
please fix build with  TCLIBC=musl

**Comment by caiotoledo** [comment]
> please fix build with TCLIBC=musl

Looking at the log provided seems that `sigval_t` is not available for musl.
But I found that you had this issue a while ago and you fixed -> https://patchwork.openembedded.org/patch/111689/

So following your approach, I'm just using `union sigval` instead of `sigval_t`.

**Review by kraj — APPROVED** [review]

**Comment by kraj** [comment]
this is now merged

**Pull request closed without merging**